app/src/main/java/com/github/andrezimmermann/gluappsample/client/mvp/event/HasClickHandlers.java

```diff
@@ -0,0 +1,10 @@
+package com.github.andrezimmermann.gluappsample.client.mvp.event;
+
+/**
+ * TODO: add javadoc
+ */
+public interface HasClickHandlers {
```

Comment:
I couldn't understand the intention of these "Has...." interfaces. It is odd to name an interface with a verb, this seems to be a good name for a method instead.